app/tests/spec/views/settings/clients.js

```diff
@@ -142,9 +142,6 @@ define(function (require, exports, module) {
       });
 
       it('renders attachedClients and apps if apps view enabled', function () {
```

Comment:
Similar to https://github.com/mozilla/fxa-content-server/blob/f8c9b1061028bc2ec8e3d2b72f696c6e5ec54552/app/tests/spec/views/settings/clients.js#L149, I think you can add a `properly sets the type of the app` that tests name and that correct place holder image is set for Pocket, Add-ons and other apps.